charts/factorio-server-charts/values.yaml

```diff
@@ -303,5 +356,17 @@ map_settings:
     - 2
     - 3
     - 4
+    negative_path_cache_delay_interval: 20
   max_failed_behavior_count: 3
-    
\ No newline at end of file
+
+admin_list:
+  # - "john_doe"
```

Comment:
These emit warnings since they don't have a value. Should specify an empty list explicitly